docs/guide/migrating-to-version-9.md

```diff
@@ -635,6 +636,10 @@ KPagination now uses [KDropdown](/components/dropdown) instead of [KSelect](/com
 * `testMode` prop has been removed
 * `onPopoverClick` prop has been removed. You can a combination of new `closeOnPopoverClick` prop and `popover-click` event instead
 * `positionFixed` prop default value has been changed to `true`
+* `placement` prop options have changed, see [Constants, Types & Interfaces](#constants-types--interfaces-15) for more details
+* `hidePopover` prop has been removed. Instead, you can use exposed by the component `hidePopover` method
+* `target` prop has been removed (popover element is always mounted to `.k-popover` element)
+* `positionFixed` prop has been removed (popover element uses `position: fixed;` as a default)
```

Comment:
```suggestion
* `positionFixed` prop has been removed (popover element always uses `position: fixed`)
```

Reply:
I think we were exposing the position-fixed somewhere else?

Reply:
Yes - removed it from all other components too